assets/js/src/newsletters/types.jsx

```diff
@@ -23,7 +30,7 @@ class NewsletterTypes extends React.Component {
     }
   };
 
-  getAutomaticEmails = () => {
+  getAutomaticEmails() {
```

Comment:
Why did you change the type of the function? I see you used `.bind(this)` below so it's OK I guess but is there any reason? Now some functions in this class are arrow functions, some are not.

Reply:
Normally all methods should be functions in order to access `this` properly. But ESLint complains about using a `function` and not using `this` inside it. So we ended up converting those to `arrow functions`. Now I changed this to be a `function` because I needed to use `this` inside.

Reply:
I see. However:
1. Arrow functions do allow you to access the correct `this` without the need to bind it.
2. It may be indeed better to use bound functions in some cases (see https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1) but in such case, the `bind()` belongs to the constructor, not as a side-effect of `render` method (https://github.com/mailpoet/mailpoet/pull/2361/files#diff-e7956c90afbfb37c0f8a5bca3e5ebfa3R95).